build/source/engine/soilLiqFlx.f90

```diff
@@ -297,27 +283,13 @@ subroutine soilLiqFlx(&
  ! preliminaries
  ! -------------------------------------------------------------------------------------------------------------------------------------------------
 
- ! define the method to compute derivatives
-
- ! numerical derivatives are not implemented yet
- if (ixDerivMethod==numerical) then
-  message=trim(message)//'numerical derivatives do not account for the cross derivatives between hydrology and thermodynamics'
+ ! validate the method to compute derivatives
+ if (ixDerivMethod/=analytical) then
+  message=trim(message)//'only analytical derivatives are available -- numerical derivatives are no longer used'  
   err=20; return
  end if
 
- ! check the need to compute analytical derivatives
- if (deriv_desired .and. ixDerivMethod==analytical) then
-  desireAnal = .true.
- else
-  desireAnal = .false.
- end if
-
- ! check the need to compute numerical derivatives
- if (deriv_desired .and. ixDerivMethod==numerical) then
-  nFlux=3  ! compute the derivatives using one-sided finite differences
- else
-  nFlux=0  ! compute analytical derivatives
- end if
+ desireAnal = .true. ! set flag for computing analytical derivatives
```

Comment:
Seems weird to set a flag to always be true. Can this flag and, if needed, any reliance on this flag later in the routine be removed?